soepy/python/simulate/simulate_auxiliary.py

```diff
@@ -18,36 +18,33 @@ def pyth_simulate(attr_dict, state_space_args, periods_emax):
         state_space_args[3],
     )
 
-    # Unpack parameter from the model specification
-    educ_min = attr_dict["INITIAL_CONDITIONS"]["educ_min"]
-    educ_max = attr_dict["INITIAL_CONDITIONS"]["educ_max"]
-    num_periods = attr_dict["GENERAL"]["num_periods"]
-    num_agents_sim = attr_dict["SIMULATION"]["num_agents_sim"]
-    seed_sim = attr_dict["SIMULATION"]["seed_sim"]
-    shocks_cov = attr_dict["DERIVED_ATTR"]["shocks_cov"]
-    optim_paras = attr_dict["PARAMETERS"]["optim_paras"]
-    delta = attr_dict["CONSTANTS"]["delta"]
-
-    educ_years = list(range(educ_min, educ_max + 1))
-    np.random.seed(seed_sim)
-    educ_years = np.random.choice(educ_years, num_agents_sim)
+    educ_years = list(range(model_params.educ_min, model_params.educ_max + 1))
+    np.random.seed(model_params.seed_sim)
+    educ_years = np.random.choice(educ_years, model_params.num_agents_sim)
 
     # Create draws for simulated sample
-    draws_sim = draw_disturbances((num_periods, num_agents_sim), shocks_cov, seed_sim)
+    draws_sim = draw_disturbances(
+        (model_params.num_periods, model_params.num_agents_sim),
+        model_params.shocks_cov,
+        model_params.seed_sim,
+    )
 
     # Start count over all simulations/rows (number of agents times number of periods)
     count = 0
 
     # Initialize container for the final output
     num_columns = 14  # count of the information units we wish to record
```

Comment:
Please move this constant to our shared_constants file. We want all constants in one central place.